.travis.yml

```diff
@@ -49,6 +52,18 @@ matrix:
 
 script: tox --recreate
 
+after_success:
+  - |
+    set -ex
+    if [[ "${TOXENV#py}" != "$TOXENV" ]]; then
+      pip install codecov coveralls
+      coverage xml
+      coverage report -m
+      coveralls
```

Comment:
Should we drop `coveralls` if we are migrating to `codecov`?

Reply:
I think it is good to keep it for comparison, at least for a while.